src/sc_crawler/description_ingestor.py

```
@@ -0,0 +1,92 @@
+    Setting the `SC_CRAWLER_DESCRIPTIONS_DATA_PATH` environment variable will
+    override the default path for persistent/cached description data access.
+    """
+    if getenv("SC_CRAWLER_DESCRIPTIONS_DATA_PATH"):
+        temp_dir = getenv("SC_CRAWLER_DESCRIPTIONS_DATA_PATH")
+        makedirs(temp_dir, exist_ok=True)
+    else:
+        temp_dir = mkdtemp()
+        register(rmtree, temp_dir)
+    zip_path = path.join(temp_dir, "downloaded.zip")
+    if not path.exists(zip_path):
+        response = get(DESCRIPTIONS_ZIP_URL)
```

Ruff (0.15.17)

[error] Probable use of `requests` call without timeout

(S113)